test/e2e/network/dns.go

```diff
@@ -80,16 +80,17 @@ var _ = SIGDescribe("DNS", func() {
 		}
 		headlessService := framework.CreateServiceSpec(dnsTestServiceName, "", true, testServiceSelector)
 		_, err := f.ClientSet.CoreV1().Services(f.Namespace.Name).Create(headlessService)
-		Expect(err).NotTo(HaveOccurred())
+		Expect(err).NotTo(HaveOccurred(), "Failed to create headless service %s", dnsTestServiceName)
```

Comment:
should be `... service: %s`
error strings should start with lowercase (applies to the rest of the commit too).
[1]